Dockerfile

```diff
@@ -38,4 +38,5 @@ COPY . /build
 # Fixes certificate errors with letsencrypt in ARMv7
 RUN echo "ca_certificate=/etc/ssl/certs/ca-certificates.crt" >> /etc/wgetrc
 RUN /build/toolchain.sh
-
+# Temporary files
+RUN rm -rf /build/build
```

Comment:
Pro tip:  This doesn't really remove the build files, it just makes them inaccessible.  Each RUN command creates a new layer which inherits the previous one in its entirety, and then places an overlay on top.

If the intention was to make the docker image smaller, a better approach would be to create a sub-image and put /build in that, and then copy the files you want to keep from the subimage to the main image after the build completes in the subimage.

Reply:
Nice tip. Thanks
Now we have a multi-stage image. Stage builder + runtime at https://github.com/ps3dev/ps3toolchain/pull/121/commits/16a02432e63d2e7a4a82cbf6490560dfa2c8e8f6
and also https://hub.docker.com/r/hldtux/ps3dev

* builder with everything needed to compile gcc
* runtime with only **$PS3DEV** content